lib/topolys/model.rb

```diff
@@ -948,50 +948,68 @@ def recalculate
           @normal = temp
         end
       end
-      
+
       raise "Cannot compute normal" if @normal.nil?
       raise "Normal has 0 length" if largest == 0
-      
+
       @normal.normalize!
-      
+
       @plane = Topolys::Plane3D.new(@directed_edges[0].v0.point, @normal)
-    
+
```

Comment:
Temporarily comparing to 10mm works for now. I think it's safe to assume third-party apps could generate inaccuracies upwards to 10mm, e.g. the 4th coordinate of an irregular 10mx10m face vs face plane (set by first 3x vertices).